DHFPROD-5623: added focus to switch view buttons in load tile

jyangML opened the pull request:
### Description
made radio button group that switches view in Load tile focusable and added outline when focused
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

wooldridge reviewed (changes requested):
@jyangML, nice work. We also want a focused switch view component to be actionable when the user presses return. I've updated the AC on the ticket to clarify this, can we add that as well?

FYI, screencast of current functionality: https://drive.google.com/file/d/1UZy7PUEhRpNPBl6DkXIhg57Cqzmlrt90/view

jyangML commented:
> @jyangML, nice work. We also want a focused switch view component to be actionable when the user presses return. I've updated the AC on the ticket to clarify this, can we add that as well?
> 
> FYI, screencast of current functionality: https://drive.google.com/file/d/1UZy7PUEhRpNPBl6DkXIhg57Cqzmlrt90/view

added the functionality you requested, also added toggle with arrow keys.
added an extra unit test in this pr as well, because I realized I forgot.

wooldridge reviewed (approved):
@jyangML, looks great! 

I can now tab to the switch element to put it in focus.
When in focus, I can use the left and right keys to switch to the left and right buttons, respectively.
When in focus, I can use the return key toggle the switch.
Unit tests test the control of the switch via the keys when element is in focus.

briantang reviewed (approved):
Looks great, and tests pass!
